### ISSUE TITLE
Views with highlightOnTouch can gain keyboard focus

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#1566]
### Problem description
Views with the `highlightOnTouch` flag enabled can gain keyboard focus just like a TextInput. Pressing on the Accept key of the keyboard while it has focus makes the ripple animation trigger on this view (no `"tap"` event is fired).

In the snippet below, the red composite will gain focus when either:
  * tapping on it, which will unfocus the text input (see "tap" handler), so the focus will jump to it
  * tapping on the "next form field" button of the keyboard

### Expected behavior
Only TextInput views should be able to gain keyboard focus. 

### Environment

- Tabris.js version: 2.2.0
- Device: Emulator
- OS: Android 8.0.0

### Code snippet

```javascript
const {Composite, TextInput, ui} = require('tabris');

let textInput;

ui.contentView.append(
  textInput = new TextInput({
    top: 10, left: '20%', right: '20%',
    keepFocus: true,
    focused: true
  }),
  new Composite({
    left: '20%', bottom: 30, right: '20%', height: 60,
    background: 'red',
    highlightOnTouch: true
  }).on({tap: () => {
    textInput.focused = false;
    console.log('composite tapped');
  }})
);
```

**Comment by mpost** [comment]
Fixed in 2.4
